data/remnant missions.txt

```diff
@@ -1713,3 +1717,151 @@ mission "Remnant: Broken Jump Drive job"
+			`	Up close, the void sprite is massive: most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It shifts through the air like a strange kind of fish and you can make out a low vibration you suspect is probably from the air whistling over it as it closes with you. Facing down this rapidly approaching giant, you feel something click in the back of your mind and a wave of paralysis washes over you. In a moment of lucidity moments before the void sprite hits, you recall that your shields are keyed to your equipment: You could activate them to protect yourself from this attack, and they would extend around you. It probably wouldn't be too healthy for the void sprite, though.`
+			choice
+				`	(Activate shields.)`
+				`	(Do not activate shields.)`
+					goto holdground
+			`	Your mind latches onto the idea and you trigger the shields in a rush. The millisecond that follows feels like an eternity: The void sprite is just a few meters away: One tendril reaches out and touches you just as the shield slams into place. You see it strike the tendril and severs it cleanly in the process of re-establishing a sealed barrier. In the same instant, the pressure on your mind vanishes, as if it was simply a dream. You momentarily stare at the immobile void sprite, both frozen in shock over what just happened. Then time reasserts itself and the sprite spasms away from the ship with an unearthly crescendo of dissonant tones. It plunges down into the clouds, leaving you standing on your hull with the floating severed piece of tendril.`
+			`	You grab the tendril and store it a sealed container. While the attack spoiled the beautiful moment, at least you have a sample and some recordings that might be of interest to Plume.`
+				flee
+			label holdground
+			`	Despite the paralysis filling your mind, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.`
```

Comment:
I'm trying to hint that the paralysis isn't internal. That the paralysis is something outside the player trying to restrict their actions. That it is, actually, something to be resisted. (and, because the game wouldn't be much fun if the player didn't get to make choices, the player manages to resist enough to make a choice here)

Reply:
OK, Github is cooperating again. I prefer to have the player resist the paralysis either way. I find that it is more dramatic to hold one's ground against such a charge deliberately, rather than simply give in to paralysis.

Reply:
Then you should really reword it somehow, because it makes no sense to me that whether you do or don't act against the void sprite, you're still somehow acting against a paralysis.

Reply:
OK. How is:
```	Despite freeing yourself from the paralysis, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.```

Reply:
@Amazinite Does that make it more clear that the player is resisting the paralysis, and *then* making a choice as to their action?